plugins/3MFWriter/SettingSelection.qml

```diff
@@ -19,7 +19,7 @@ RowLayout
         Layout.preferredWidth: UM.Theme.getSize("setting").width
         checked: modelData.selected
         onClicked: modelData.selected = checked
-        enabled: modelData.selectable
+        tooltip: modelData.selectable ? "" :catalog.i18nc("@tooltip", "This setting may not perform well while exporting in UCP. Users are asked to add it at their own risk")
```

Comment:
... exporting in UCP. -> .. exporting _to_ UCP.

Users are asked to add it at their own risk -> Users are advised to add it at their own risk only.

(Also note the missing period at the end as well.)